src/GaussianDistributions.jl

```diff
@@ -59,9 +59,11 @@ whiten(Σ::UniformScaling, z) = z/sqrt(Σ.λ)
 sqmahal(P::Gaussian, x) = norm_sqr(whiten(P.Σ,x - P.μ))
 
 rand(P::Gaussian) = P.μ + chol(P.Σ)'*randn(typeof(P.μ))
-rand(P::Gaussian{Vector}) = P.μ + chol(P.Σ)'*randn(T, length(P.μ))
+rand(P::Gaussian{V}) where V<:AbstractVector{T} where T =
```

Comment:
`V<:AbstractVector{T}` would catch `SVector`'s, but those have to call `randn(typeof(P.μ))`

Reply:
Oh, that's neat... I'll fix it.